src/common/utils/validator.js

```diff
@@ -212,6 +218,14 @@ function currentOrPastDate(dObj) {
     return true;
   }
 
+  /*
+    Returns true if supplied dates are invalid to avoid duplicate error messages being displayed.
+    The dateNotTooFarInFuture function will cover this error.
+  */
+  if(validDay(dObj.d, dObj.m, dObj.y) == false || validMonth(dObj.m) == false || validYear(dObj.y) == false){
```

Comment:
Please stop using `==` or `!=` in javascript code.
* By removing strict type checking from comparsion, it remove information of what `validDay`, `validMonth` or `validYear` return e.g. is it returning the year number (e.g. `2024`) or `false` boolean.

It is not a issue here, but if done in other parts of the code, it can lead to hard to find bugs.
* There does not seem be a good reason to use loose equality here.

Reply:
Has been done, thanks for doing so.